src/routes/private.ts

```diff
@@ -67,6 +69,18 @@ export default async function (fastify: FastifyInstance): Promise<void> {
     const operationResult = await addNewUser(con, req.body, req.log);
     return res.status(200).send(operationResult);
   });
+  fastify.post<{ Body: UpdateUserEmailData }>(
+    '/updateUserEmail',
+    async (req, res) => {
+      if (!req.service) {
+        return res.status(401).send();
```

Comment:
We usually return `404` so an attacker will not know this endpoint exists even
```suggestion
        return res.status(404).send();
```